src/libraries/System.Private.CoreLib/src/System/Runtime/InteropServices/PlatformAttribute.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Runtime.InteropServices
+{
+
+    /// <summary>
+    /// Base type for all platform-specific attributes. Primarily used to allow grouping
+    /// in documentation.
+    /// </summary>
+    public abstract class PlatformAttribute : Attribute
+    {
+        internal PlatformAttribute(string platformName)
```

Comment:
internal?

Reply:
Yeah, that's what we picked in the review.  It could also say `private protected`; but since all of the specific attributes are sealed we decided to make the family assembly-locked, so it didn't give a false sense of extensibility.

Reply:
Yes, it should be `private protected`